### ISSUE TITLE
Popover or actions popover should be full screen on mobile

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/nextcloud-vue#3159]
From discussions with @jancborchardt we should consider moving to a full screen popover when the space is to small as the current automatic position is not very nice sometimes then.

#3157 adds a height limit and automatic overflow behaviour but is only a first step.

One idea would be to have a bottom drawer like it is common on android for example https://material.io/components/navigation-drawer#bottom-drawer which might also be a relevant topic in terms of accessibility.
